test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -9977,7 +9977,7 @@ public void RestoreNetCore_WithMultipleFrameworksWithPlatformAndAssetTargetFallb
 
                 var frameworks = new NuGetFramework[]
                 {
-                    NuGetFramework.Parse("net5.0-windows10.0.10000.1"),
+                    NuGetFramework.Parse("net5.0-windows10.0.6000"),
```

Comment:
aren't we supposed to keep supporting 4-part versions?

Reply:
It was my test infra that was lacking. I've reverted it.